### PR TITLE
feat(scully): logrocket render plugin.

### PR DESCRIPTION
**Pull request by GuzmanPI**
The docs app uses this plugin.

Implements the pluguin

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/scullyio/scully/blob/main/CONTRIBUTING.md#commit
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Other... Please describe:

LogRocket plugin for the docs. This may be an npm plugin in the future.

## What is the current behavior?

Issue Number: #673 

## What is the new behavior?

Adds LogRocket's scripts to the <head> tag.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: libs/plugins/logrocket/src/lib/plugins-logrocket.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import { registerPlugin, getMyConfig, HandledRoute } from '@scullyio/scully';
+
+export const LogRocket = 'logrocket';
+
+const logrocketPlugin = async (
+  html: string,
+  route: HandledRoute
+): Promise<string> => {
+  const logrocketConfig = getMyConfig('logrocket');
```
Comment (SanderElias) [inline libs/plugins/logrocket/src/lib/plugins-logrocket.ts:9]:
```suggestion
  const logrocketConfig = getMyConfig(logrocketPlugin);
```
Works better with just the plugin itself

**Review by SanderElias — APPROVED** [review]
Please change the thing I highlighted!
Also, shouldn't you define a default config in there?

BTW, you don't need to blush it to be used in scully.

File: libs/plugins/logrocket/src/lib/plugins-logrocket.ts (hunk 1 above)
Comment (GuzmanPI) [reply libs/plugins/logrocket/src/lib/plugins-logrocket.ts:9]: I noticed you used the plugin itself in the bas-href-rewrite, but I was not sure how to do it. Thx @SanderElias!

Comment (GuzmanPI) [reply libs/plugins/logrocket/src/lib/plugins-logrocket.ts:9]: I think more than a default config, it should throw an error if there is no config. I'll make the changes

**Comment by jorgeucano** [comment]
@GuzmanPI I check the test and show this error

```    +     </style><script>window['ScullyIO']='generated';</script>
    +     <script src="https://cdn.logrocket.io/LogRocket.min.js"></script>
    +     <script>window.LogRocket && window.LogRocket.init('undefined/undefined');</script></head>``` 
Please check the config is not working!

**Comment by GuzmanPI** [comment]
I'll check the tests

**Pull request merged**